swarm_examples/README.md

```diff
@@ -141,35 +146,39 @@ The swarm can be started with the similar command but note that the process will
 *take longer* as it has to try a bunch of combinations.
 
 ```
-%> run_swarm.py multi1_search_def.json --overwrite --maxWorkers 5
+%> $NUPIC/scripts/run_swarm.py multi1_search_def.json --overwrite --maxWorkers 5
 ```
 
 In the run I did, I got the following error:
 
 ```
 Best results on the optimization metric multiStepBestPredictions:multiStep:errorMetric='altMAPE':steps=[1]:window=1000:field=metric1 (maximize=False):
-[52] Experiment _GrokModelInfo(jobID=1161, modelID=23650, status=completed, completionReason=eof, updateCounter=22, numRecords=1500) (modelParams|clParams|alpha_0.0248715879513.modelParams|tpParams|minThreshold_10.modelParams|tpParams|activationThreshold_13.modelParams|tpParams|pamLength_2.modelParams|sensorParams|encoders|metric2:n_271.modelParams|sensorParams|encoders|metric1:n_392.modelParams|spParams|synPermInactiveDec_0.0727958344423):
-  multiStepBestPredictions:multiStep:errorMetric='altMAPE':steps=[1]:window=1000:field=metric1:    0.886040768868
+[9] Experiment _NupicModelInfo(jobID=1458, modelID=124313, status=completed, completionReason=eof, updateCounter=22, numRecords=1500) (modelParams|clParams|alpha_0.1.modelParams|tpParams|minThreshold_12.modelParams|tpParams|activationThreshold_16.modelParams|tpParams|pamLength_5.modelParams|sensorParams|encoders|metric1:n_521.modelParams|spParams|synPermInactiveDec_0.005285):
+  multiStepBestPredictions:multiStep:errorMetric='altMAPE':steps=[1]:window=1000:field=metric1:    2.16445919912
 ```
 
-The end error was 0.89%, significantly better than before. This means at least one 
-additional field helped.  But which one? At the end, look for a Field Contributions 
-JSON, that looks like this:
+Previous:
+> The end error was 0.89%, significantly better than before. This means at least
+> one additional field helped.  But which one? At the end, look for a Field 
+> Contributions JSON, that looks like this:
+
+I'm not sure what happened here. -- Matt
```

Comment:
with basic_search_def.json I got an error of 2.15, so the above error of 1.08 is still a significant improvement.